### PR TITLE
Australia region fixes

### PR DESCRIPTION
**Pull request by dfrencham**
- Add Australian SKUs
- Fix spelling of Australia in filename
- Improve output formatting
- Change timestamp to use current user locale
- Change store list markdown to Store Name order

### CODE REVIEW

File: constants.js
Comment (worthbak) [inline constants.js:44]: nice - probably could remove the comment on line 39 that notes issues with AU stock

File: index.js
Comment (worthbak) [inline index.js]: this is a great idea, to swap out the US favorite SKUs with the corresponding AU ones, but it might confuse people if they customize their SKUs in the `favorites` declaration, only to have them swapped out here. not sure what the best way to handle this would be - perhaps some sort of US -> AU SKU mapping? or perhaps we just leave it alone and leave it up to users to get their favorite SKUs correct?

File: index.js
Comment (worthbak) [inline index.js]: does adding a new line here create some risk that the text will be cut off when the macOS notification appears? i don't know how tall those things can get.

File: apple-store-numbers-australia.md
Comment (worthbak) [inline apple-store-numbers-australia.md:3]: thanks for correcting these 👍

**Review by worthbak — COMMENTED** [review]
thanks so much for figuring this out! hadn't spent a ton of time looking at it - good to know that it was just the fact that Australia uses different SKUs than other countries. a couple questions on this before we merge 👍

File: constants.js
Comment (dfrencham) [reply constants.js:44]: Done :)

File: index.js
Comment (dfrencham) [reply index.js]: I've relocated it to the top of the file (line 10) and added a comment - would this reduce confusion?

I like the idea of a SKU mapping, but they may not always have a 1 to 1 mapping (plus it is one more thing to maintain).

File: index.js
Comment (dfrencham) [reply index.js]: Good question! I'm testing under Windows 10 so I'm not sure. I've updated the code so that the original pipe separator is used in the notification, but the line break is used in console output. Best of both worlds ? :)

Comment (worthbak) [reply index.js]: yup, sounds good to me!

File: index.js
Comment (worthbak) [reply index.js]: yeah, i think that'll have to do for now - not ideal, but prob the best we can do.

**Review by worthbak — APPROVED** [review]

**Pull request merged**
